src/fides/api/schemas/connection_configuration/connection_secrets_bigquery.py

```diff
@@ -38,8 +38,12 @@ class BigQuerySchema(ConnectionConfigSecretsSchema):
         title="BigQuery Dataset",
         description="The dataset within your BigQuery project that contains the tables you want to access.",
     )
+    project: List[str] = Field(
+        title="BigQuery projects",
+        description="The projects within your BigQuery organization that you want to scope your connection to.",
+    )
```

Comment:
i'm actually going to remove this altogether, it's not being used currently and i just had forgotten to clean it up. this will better belong on the monitor config in any case, when we want to bring it in.